src/main/java/com/tictactoe/Player.java

```diff
@@ -3,4 +3,6 @@
 public interface Player {
     String getSymbol();
     int obtainValidCellSelection();
+    void addBoard(Board board);
```

Comment:
Again, the `addBoard` method seems a bit strange to me for a `Player`. If a method needed to use a board, I would probably pass it in via an argument rather than include it in the `Player` interface.

This might be a personal preference though. Maybe @vincentstorme agrees/disagrees?